src/integrationTest/java/uk/gov/hmcts/darts/testutils/stubs/EventStub.java

```diff
@@ -38,4 +38,19 @@ public EventEntity createEvent(HearingEntity hearing, int eventHandlerId) {
         return eventEntity;
     }
 
+    public EventEntity createEvent(HearingEntity hearing, int eventHandlerId, OffsetDateTime eventTimestamp) {
```

Comment:
maybe could combine this with existing createEvent method?

Reply:
I've amended the method so the others call the newly created one